Assets/_Scripts/Controllers/PlayerController.cs

```diff
@@ -0,0 +1,40 @@
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+//ensures that these components are attached to the gameobject
+[RequireComponent(typeof(Rigidbody2D), typeof(SpriteRenderer), typeof(Animator))]
+public class PlayerController : MonoBehaviour
+{
+    public float moveSpeed = 5f; // Speed of player movement
+    public GameObject bulletPrefab; // Prefab of the bullet GameObject
+    public Transform firePoint; // Point where the bullet will be spawned
+    public float bulletSpeed = 10f; // Speed of the bullet
```

Comment:
Agree with Mark, setting these to private means they aren't accessed by classes outside of this one and adding `[SerializeField]` means we can update them from the unity editor which will be useful when tweaking things to make them look right next week

Reply:
all have been set to [SerializeField] and most of the public has been changed to private